bionic/dagviz.py

```diff
@@ -14,6 +14,26 @@
 Image = import_optional_dependency('PIL.Image', purpose=module_purpose)
 
 
+class SvgImage(object):
+    def __init__(self, xml_string):
+        self.xml_string = xml_string
+
+    def save(self, fp):
+        '''
+        Save SVG to a file
+        '''
+        with open(fp, 'wb') as file:
+            file.write(self.xml_string)
+
+    def _repr_svg(self):
+        '''
+        Rich display SVG in IPython notebook or Qt console
+        '''
+        return self.xml_string
+
+    def show(self):
```

Comment:
Like you said, it's not clear how badly we need this. (I guess people using text-based shells like IPython might use this?) I wouldn't go out of your way to make this work.